README.md

```diff
@@ -161,6 +161,22 @@ But here are some of the more useful ones:
 
 See explanation in the top of this document.
 
+## I saw it, but I miss `.plexupdate`
```

Comment:
What do you think about moving this type of stuff into a wiki page and just leaving the README as a "here's how you use plexupdate" document?